kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-case-mgmt/src/main/java/org/kie/server/remote/rest/casemgmt/CaseAdminResource.java

```diff
@@ -79,10 +79,9 @@ public CaseAdminResource(
         this.caseAdminServiceBase = caseAdminServiceBase;
     }
 
-    @ApiOperation(value="Returns case instances without authentication checks.",
-            response=CaseInstanceList.class, code=200)
+    @ApiOperation(value="Returns case instances without authentication checks.")
     @ApiResponses(value = { @ApiResponse(code = 500, message = "Unexpected error"), 
-            @ApiResponse(code = 200, message = "Successfull response", examples=@Example(value= {
+            @ApiResponse(code = 200, response = CaseInstanceList.class, message = "Successfull response", examples=@Example(value= {
```

Comment:
Sorry about @fjtirado - seems you changed this successful message for all modules except for this case-mgmt one. Do you mind updating this as well?
Thanks again